packages/server/lib/controllers/config.controller.ts

```diff
@@ -284,9 +284,9 @@ class ConfigController {
             }
 
             const config = await configService.getProviderConfig(providerConfigKey, environmentId);
-            const environmentUuid = await environmentService.getAccountUUIDFromEnvironment(environmentId);
+            const environment = await environmentService.getById(environmentId);
```

Comment:
not entirely sure if it never worked or we just got lucky?

Reply:
What do you mean? Are you seeing an incorrect value with how it was previously?

Reply:
Ah ok I see, thanks. That value isn't used but rather the frontend uses the webhook_receive_url from the environment.